irc.html

```diff
@@ -64,7 +64,65 @@ <h1 class="display-2 pb-3" style="line-height: 1.2;">International Research Coun
     <div id="researchers" class="container">
         <h1 class="display-3 text-center mb-5">Researchers</h1>
         <!--Profile cards goes here-->
-        <div id="teamContent"></div>
+        <div id="teamContent">
+          <!-- Skeleton Profiles goes here -->
+          <div class="container">
+            <div class="row"> 
+                <div class="col-md-3">
+                    <div class="skeleton">
+                        <div>
+                            <div class="row" style=" justify-content: center">
+                                <div class="avatar"></div>
+                            </div>
+                            <div class="row profile-name ">
+                                <div class="line"></div>
+                                <div class="line line-secondary"></div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-3">
+                    <div class="skeleton">
+                        <div>
+                            <div class="row" style=" justify-content: center">
+                                <div class="avatar"></div>
+                            </div>
+                            <div class="row profile-name ">
+                                <div class="line"></div>
+                                <div class="line line-secondary"></div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-3">
+                    <div class="skeleton">
+                        <div>
+                            <div class="row" style=" justify-content: center">
+                                <div class="avatar"></div>
+                            </div>
+                            <div class="row profile-name ">
+                                <div class="line"></div>
+                                <div class="line line-secondary"></div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-3">
+                    <div class="skeleton">
+                        <div>
+                            <div class="row" style=" justify-content: center">
+                                <div class="avatar"></div>
+                            </div>
+                            <div class="row profile-name ">
+                                <div class="line"></div>
+                                <div class="line line-secondary"></div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        </div>
```

Comment:
Is this dev properly closed??

Reply:
Yes, just some indentation problem. Fixed it.